pkg/server/claims.go

```diff
@@ -177,31 +175,39 @@ func parsePrivateClaims(oidcToken jwt.Token) (*oidcClaims, error) {
 	// Use this string split instead of attempting to use this and the repository_owner claim since
 	// the repository_owner claim is optional.
 	repoParts := strings.Split(claims.Repository, "/")
-	if len(repoParts) != 2 {
-		return nil, fmt.Errorf("'repository' claim formatted incorrectly, requires <org_name>/<repo_name> format - received [%s]", claims.Repository)
+	if len(repoParts) == 2 {
+		claims.ParsedOrgName = repoParts[0]
+		claims.ParsedRepoName = repoParts[1]
+	} else {
+		claims.ParsedOrgName = ""
+		claims.ParsedRepoName = ""
```

Comment:
Could prefer to just set these to empty string, then the if statement just overrides it if its len is correct